src/lib/admin-page/index.js

```diff
@@ -0,0 +1,19 @@
+wp.domReady( () => {
+	const { sprintf } = wp.i18n;
+	const keyEl = document.querySelector( '#apikey' );
+	const requiresRecrawlNotice = document.querySelectorAll(
+		'div.parsely-form-controls[data-requires-recrawl="true"] .help-text'
+	);
+	if ( ! ( keyEl && requiresRecrawlNotice.length ) ) {
+		return;
+	}
+
+	const notice = sprintf(
+		'<p class="description"><strong style="color:red;">Important:</strong> changing this value on a site currently tracked with Parse.ly will require reprocessing of your Parse.ly data. Once you have changed this value, please contact <a href="mailto:support@parsely.com?subject=Please reprocess %s">support@parsely.com</a></p>',
```

Comment:
```suggestion
		'<p class="description"><strong style="color:#d63638;">Important:</strong> changing this value on a site currently tracked with Parse.ly will require reprocessing of your Parse.ly data. Once you have changed this value, please contact <a href="mailto:support@parsely.com?subject=Please reprocess %s">support@parsely.com</a></p>',
```

Just changes the red slightly to match the notice/form error red colour that is in the default admin colour scheme (aiming for consistency). Here's a comparison (new red is second instance):
<img width="325" alt="Screenshot 2021-04-23 at 10 14 43" src="https://user-images.githubusercontent.com/88371/115849674-059b4200-a41d-11eb-9a7e-29002810de7a.png">

Alternatively, leave as `red` for now, and we can reassess this when it's refactored to remove the noise (since technically this "Important" isn't an error).

Reply:
I'm definitely good with this change, but I'd pull it out to a separate PR.

The color value is coming from here:
https://github.com/Parsely/wp-parsely/blob/1a398fac2ebc4935199299977522c5a817b62505/parsely-admin-header.php#L32

Reply:
Opened https://github.com/Parsely/wp-parsely/issues/254.